packer.mk

```diff
@@ -29,10 +29,7 @@ else
 	$(error HYPERV_GENERATION was invalid ${HYPERV_GENERATION})
 endif
 ifeq (${OS_SKU},Ubuntu)
-ifeq (${IMG_SKU},20_04-lts-cvm)
```

Comment:
After the PR https://github.com/Azure/AgentBaker/pull/5797 was merged, this PR should be rebased to get the latest master.